src/content/docs/usage/installation.md

```diff
@@ -83,86 +70,86 @@ The main steps will be the following:
 
 The step to install Nextflow in itself will afterwards be the same as previously mentioned.
 
-### Edge releases
+### Updating Nextflow
 
-Stable releases will become less frequent as Nextflow shifts to a more dynamic development model with the use of plugins. This will allow functionality to be added as an extension to the core codebase with a release cycle that is potentially independent of Nextflow itself. As a result of the reduction in stable releases, some pipelines may require the use of Nextflow `edge` releases in order to exploit cutting "edge" features. For example, version 3.0 of the nf-core/rnaseq pipeline requires Nextflow `>=20.11.0-edge`, which allows Singularity containers to be downloaded over `http` (see [nf-core/rnaseq#496](https://github.com/nf-core/rnaseq/issues/496)).
+We recommend using a personal installation of Nextflow where possible, instead of using a system-wide installation. This makes it easier to update.
```

Comment:
This would a bit late at this point, they would have to reinstall from scratch if initially they had asked a sys admin for a global one...

Reply:
I'll move it up 👍🏻